src/main/java/blackjack/domain/card/Card.java

```diff
@@ -0,0 +1,68 @@
+package blackjack.domain.card;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Card {
+
+	// 52장의 Card를 미리 Pool에 생성
+	private static final Map<String, Card> pool = Arrays.stream(Suit.values())
```

Comment:
```suggestion
	private static final Map<String, Card> POOL = Arrays.stream(Suit.values())
```
static 상수라면 대문자로 사용해주세요
pool 보다는 조금 더 의미 있는 네이밍을 해주면 좋을것 같네요! 적어도 CARD_POOL 정도?

Reply:
> static 상수라면 대문자로 사용해주세요 pool 보다는 조금 더 의미 있는 네이밍을 해주면 좋을것 같네요! 적어도 CARD_POOL 정도?

앗..!대문자 처리를 놓쳤네요 🥲
확인 감사드립니다!

CARD_POOL 네이밍도 너무 좋은 것 같습니다!